lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -544,14 +544,14 @@ class ReportUIFeatures {
 
   /**
    * Opens a new tab to the online viewer and sends the local page's JSON results
-   * to the online viewer using postMessage.
+   * to the online viewer using URL.fragment
    * @param {LH.Result} json
    * @protected
    */
-  static openTabAndSendJsonReportToViewer(json) {
+  static openViewer(json) {
     const windowName = 'viewer-' + this.computeWindowNameSuffix(json);
     const url = getAppsOrigin() + '/viewer/';
-    ReportUIFeatures.openTabAndSendData({lhr: json}, url, windowName);
+    ReportUIFeatures.openTabWithUrlData({lhr: json}, url, windowName);
```

Comment:
do we want this fragment stuff to become the standard?

Reply:
@exterkamp is about to find out if we can use the postMessage technique with LB. I don't see any reason why it would be a problem, but.. we'll find out what happens…

Reply:
resolved now? AFAIK we're following the logic as of  your L505